### PR TITLE
Add changelog, bump version, update gemspec.

### PR DESCRIPTION
**Pull request by djberg96**
Like the title says:

* Adds a CHANGELOG.md
* Bumps the version to 0.1.1
* Adds the CHANGELOG.md to the gemspec files

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## 0.1.1 - May 4, 2020
+### Changed
```
Comment (bdunne) [inline CHANGELOG.md:2]: This also includes #4 I'm not sure if that changes the version number

**Comment by bdunne** [comment]
Also, we usually don't make this a PR.  Usually it's just "Release vX.Y.Z"

File: CHANGELOG.md (hunk 1 above)
Comment (djberg96) [reply CHANGELOG.md:2]: Updated. No, it doesn't change the version number.

**Comment by djberg96** [comment]
@bdunne I normally would just make it a release, but since I added the changelog, I wasn't sure if I should.

**Pull request merged**